server/models/video/video-import.ts

```diff
@@ -161,6 +161,23 @@ export class VideoImportModel extends Model<Partial<AttributesOnly<VideoImportMo
     ]).then(([ total, data ]) => ({ total, data }))
   }
 
+  static async urlAlreadyImported (userId: number, targetUrl: string): Promise<boolean> {
```

Comment:
I think you need to check if the video has been imported in the channel, and not by the user because we may import the same video in multiple channels

Reply:
I think I encountered some difficulties doing that. I'll try again and keep you informed of them if any.

Reply:
Done 1de2b573a33b4792623ec1616ed7df78e5295d11